src/internal/pachsql/table_info.go

```diff
@@ -0,0 +1,82 @@
+package pachsql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+)
+
+// TableInfo contains information about a SQL table
+type TableInfo struct {
+	Columns []ColumnInfo
+}
+
+// ColumnInfo is information about a single column in a SQL table
+type ColumnInfo struct {
+	TableSchema string
+	TableName   string
```

Comment:
feels weird modeling-wise to have these in every column instead of in `TableInfo`